conf/nginx-proxy.conf

```diff
@@ -8,6 +8,9 @@ location YNH_LOCATION {
   proxy_set_header  X-Forwarded-Host $server_name;
   proxy_set_header  X-Forwarded-Port $server_port;
   
+  #allow downloading files >1GB
+  proxy_buffering off;
```

Comment:
WIth this options the include of the yunohost square won't work i think...

Reply:
```suggestion
-  proxy_buffering off;
+ client_max_body_size 10G;
```

Reply:
In that case, I suggest considering a larger limit, like 100G, larger than a Blu-ray.